Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -154,8 +154,6 @@ end OmegaCompletePartialOrder
 
 open OmegaCompletePartialOrder
 
--- Porting note: removed "set_option extends_priority 50"
```

Comment:
This is a bit of a regression but I suppose this will be very dependent on a better implementation of default instance priorities anyway.

Reply:
Yes, I think fixing this is a big change, that we'll hopefully eventually tackle, and will require a thorough review (or at least thorough benchmarking) everywhere.